go.mod

```diff
@@ -3,33 +3,33 @@ module sigs.k8s.io/cluster-api-provider-azure
 go 1.13
 
 require (
-	github.com/Azure/azure-sdk-for-go v44.0.0+incompatible
-	github.com/Azure/go-autorest/autorest v0.11.0
+	github.com/Azure/azure-sdk-for-go v44.2.0+incompatible
```

Comment:
they actually released v0.45.0 since you opened this

Reply:
will update

Reply:
done